packages/disputer/src/disputer.js

```diff
@@ -122,40 +124,42 @@ class Disputer {
           this.logger.error({
             at: "Disputer",
             message: "Cannot dispute: price feed returned invalid value",
-            error: error.message
```

Comment:
We can't assume that `error.message` exists because some of the PriceFeeds pass through arrays of errors such as the `BasketSpreadPriceFeed`.

Reply:
TODO: Either combine error messages within the PriceFeeds or handle arrays of errors from the Logger.

Reply:
Would toString work? Or would we lose info?

Reply:
i think we migth need to change how the `Logger` handles error objects: https://github.com/UMAprotocol/protocol/blob/master/packages/financial-templates-lib/src/logger/Logger.js#L43